dnanexus/src/eggd_athena.sh

```
@@ -26,11 +26,19 @@ _set_string_inputs() {
     '''
-    [ -n "$panel" ] && panel="--panel '${panel}' " || unset panel
+    [ -n "$panel" ] && panel_arg="--panel '${panel}' " || unset panel
     [ -n "$panel_filters" ] && panel_filters="--panel_filters ${panel_filters} " || unset panel_filters
     [ -n "$indication" ] && indication="--indication '${indication}' " || unset indication
 }
 
+_set_optional_file_inputs() {
+    : '''
+    Format optional file inputs correctly
+    '''
+    [ -n "$normal_coverage" ] && normal_coverage="--normal_coverage ${normal_coverage_path}" || unset normal_coverage
```

Shellcheck (0.10.0)

[warning] normal_coverage_path is referenced but not assigned.

(SC2154)